File: common/src/main/java/com/thesilentnights/pojo/MailAccountEntry.java
package com.thesilentnights.pojo;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class MailAccountEntry {
    private String user;
    private String password;
    private String host;
    private int port;
    private String from;
    private boolean enableSSL;
    private boolean starttlsEnable;
    private long timeout;
}

File: src/main/kotlin/com/thesilentnights/easylogin/pojo/MailAccountEntry.kt
package com.thesilentnights.easylogin.pojo


data class MailAccountEntry(
    var username: String?,
    var password: String?,
    var host: String?,
    var port: Int = 0,
    var from: String?,
    var enableSSL: Boolean = false,
    var starttlsEnable: Boolean = false,
    var timeout: Long = 0,
)